test/test_menuMapper.js

```diff
@@ -0,0 +1,42 @@
+var assert = require('assert');
+var mapper = require('../mappers/menuMapper');
```

Comment:
Depending  on whether everyone agrees with my changes to the DB testing (Running these tests just wiped my DB again >:o ) Then we could reflect this here. Should be only minor changes!